test/test_utils/__init__.py

```diff
@@ -43,6 +43,9 @@
 # Reason string for skipping tests in non-PR context
 PR_ONLY_REASON = "Skipping test that doesn't need to be run outside of PR context."
 
+#Reason string for skipping tests temporarily
+TEMP_REASON = "Skipping test for temporary reason"
```

Comment:
No code change needed: We might be able to use a more descriptive phrase for this temp reason, or just use a temp string instead of a variable with an unobvious phrase.

Reply:
+1